setup.py

```diff
@@ -84,7 +84,7 @@ def get_git_sha() -> str:
         "deprecation>=2.1.0, <2.2.0",
         "flask>=2.2.5, <3.0.0",
         "flask-appbuilder>=4.3.9, <5.0.0",
-        "flask-caching>=1.11.1, <2.0",
+        "flask-caching>=2.0, <3",
```

Comment:
Minor comment: I think it's been customary to set the lower bound to the version we're locking to in the requirements. I would maybe consider doing the following, unless there's strong feelings about the need for supporting `>=2.0.0, <2.1`
```suggestion
        "flask-caching>=2.1.0, <3",
```

Reply:
Plus one on @villebro's suggestion.